.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python:
   - "2.7"
-script: python test.py
+script:
+  - find . -name '*.py' -print0 | xargs -0 flake8
```

Comment:
Thanks, I knew there had to be an easier way to do this. Scrolling up into my shell history, I only tried `flake8 *` and forgot about `flake8 .`.